src/assets/app.extensions.json

```diff
@@ -8,6 +8,7 @@
   "$runtime": "1.7.0",
   "$description": "Core application extensions and features",
   "$references": [],
+  "$ignoreReferenceList": ["about.plugin.json", "settings.plugin.json"],
```

Comment:
can we use the APP_EXTENSIONS_IGNORE_REFS variable directly here so we don't need the extra shell script? and add a replacer in the startup for the default value

Reply:
not everything runs via docker, using variable there will break the whole development process

Reply:
the replacer like you have in the other projects supports plain javascript runs maybe @popovicsandras can explain better